### PR TITLE
Removes special handling of non-LD QA results.

### PR DESCRIPTION
**Pull request by justinlittman**
QA is switching to use the same context format in non-LD result as LD results.

### CODE REVIEW

**Comment by justinlittman** [comment]
Pointing at QA staging (which has the context changes):
image

**Comment by justinlittman** [comment]
To test this, you can change https://lookup.ld4l.org/authorities to https://lookup-stg.ld4l.org/authorities in `apidoc.json`.

The release of this to production will be coordinated with the release of QA server containing the accompanying changes.

File: src/lib/apidoc.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   },
   "servers": [
     {
-      "url": "https://lookup.ld4l.org/authorities",
+      "url": "https://lookup-stg.ld4l.org/authorities",
```
Comment (jermnelson) [inline src/lib/apidoc.json]: Should this be reverted back to the production QA URI?

Comment (justinlittman) [reply src/lib/apidoc.json]: Umm, definitely.

**Pull request merged**